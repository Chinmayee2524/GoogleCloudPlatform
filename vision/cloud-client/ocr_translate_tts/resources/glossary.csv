Foobar,Foobar
Product name,Product name