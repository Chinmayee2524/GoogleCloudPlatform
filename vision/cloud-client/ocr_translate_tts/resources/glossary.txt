FooBar is a Foo, but not a Bar.
I love this Product Name.