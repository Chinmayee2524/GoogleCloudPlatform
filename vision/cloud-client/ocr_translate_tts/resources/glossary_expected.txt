FooBar est un Foo, mais pas un bar. J&#39;aime ce Product Name.